tanner/create_tables.py

```diff
@@ -0,0 +1,71 @@
+import asyncio
+
+
+class CreateTables:
+    @staticmethod
+    async def create_data_tables(pg_client):
+        """Create all the required tables in
+            the postgres database
+
+        Arguments:
+            pg_client {aiopg.pool.Pool}
+        """
+        async with pg_client.acquire() as conn:
+            async with conn.cursor() as cur:
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "paths" (
+                    "sess_uuid" UUID PRIMARY KEY NOT NULL UNIQUE,
+                    "path" TEXT NOT NULL,
+                    "timestamp" TIMESTAMP NOT NULL,
+                    "response_status" INT NOT NULL,
+                    "attack_type" INT NOT NULL
```

Comment:
Create an additional table with attack types. Integer id and a string with description (e.g. sqli, xss...) and the add a foreign key constraint here.

Reply:
@afeena  suggested that we can have a dictionary which resolves the attack type to an integer something like:
```json
{
	"sqli":1,
	"xss":2,
}
```
And in DB we can store the integer, since it will be cheaper to store.

Reply:
yeah, same suggestions except in mine the dictionary will be in a table as well. On startup you can fetch that table and populate your in memory dictionary.

Reply:
I thought about this table in the beginning when I started to think about schema. But then I realise that we create attack type table from tanner and convert types inside tanner to put in into the table and then we fetch it from table... For me, it's a little bit redundant.

I prefer to have the Enum class in tanner which represents attack types and we can easily do conversion inside tanner. But if you think it would be better to have a table, I have no complains

Reply:
From the other hand, it would be easier to fetch only attack types if we want to do analysis only on that level, so in this case I agree that separate table is much more convenient

Reply:
No, sounds good to me. Adding that table later would be easy 👍

Reply:
@glaslos Sure, if on the stage of requests implementation it would be easier to have separate table, we always can create it :)